tests/agent/test_python.py

```diff
@@ -7,8 +7,13 @@
 @pytest.mark.version
 @pytest.mark.flask
 def test_req_flask(flask):
-    utils.check_agent_transaction(
-        flask.foo, flask.apm_server.elasticsearch, ct=2)
+    utils.check_agent_transaction(flask.foo, flask.apm_server.elasticsearch)
+
+
+@pytest.mark.version
+@pytest.mark.flask
+def test_flask_error(flask):
+    utils.check_agent_error(flask.oof, flask.apm_server.elasticsearch, ct=2)
```

Comment:
@beniwohli I was surprised this generated two documents

Reply:
I haven't looked at the actual documents, but this is probably due to [`logging=True`](https://github.com/elastic/apm-integration-testing/pull/203/files#diff-b4c662e3253b95a7af5361efeee2512dR24), which automatically sets up logging using our `LoggingHandler`. So one error is from our exception handler, and the other one from the logging handler, at least if my guess is right.

Reply:
Looks right based on the documents produced, good with that for the tests?